src/modules/volumes/volume-list/volume-list.js

```diff
@@ -87,25 +90,78 @@
                 });
         }
 
-        function _compareFn(item1, item2) {
-            var compValue = 0;
-            if (vm.sortConfig.currentField.id === "name") {
-                compValue = item1.name.localeCompare(item2.name);
-            } else if (vm.sortConfig.currentField.id === "status") {
-                compValue = item1.status.localeCompare(item2.status);
-            }
+        function disableProfiling(volume, $event) {
 
-            if (!vm.sortConfig.isAscending) {
-                compValue = compValue * -1;
+            volumeStore.disableProfiling(volume, vm.clusterId)
```

Comment:
same as above,

create a single and generic function for both enable and disable